src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -137,6 +160,9 @@ class SGD
   //! The instantiated function.
   DecomposableFunctionType& function;
 
+  //! The update policy used to update the parameters in each iteration.
+  UpdatePolicyType updatePolicyType;
```

Comment:
I think we could use ``updatePolicy``instead of ``updatePolicyType``, it's not a datatype but it's a policy type, so I think it also works. What do you think?